test/units/cli/test_galaxy.py

```diff
@@ -444,3 +450,123 @@ def test_template_ignore_similar_folder(self):
 
     def test_skeleton_option(self):
         self.assertEquals(self.role_skeleton_path, context.CLIARGS['role_skeleton'], msg='Skeleton path was not parsed properly from the command line')
+
+
+@pytest.fixture()
+def collection_skeleton(request):
+    name, skeleton_path = request.param
+
+    galaxy_args = ['ansible-galaxy', 'collection', 'init', '-c']
+
+    if skeleton_path is not None:
+        galaxy_args += ['--collection-skeleton', skeleton_path]
+
+    test_dir = tempfile.mkdtemp()
+    if not os.path.isdir(test_dir):
+        os.makedirs(test_dir)
+    galaxy_args += ['--init-path', test_dir, name]
+
+    gc = GalaxyCLI(args=galaxy_args)
+    gc.run()
+
+    namespace_name, collection_name = name.split('.', 1)
+    collection_dir = os.path.join(test_dir, namespace_name, collection_name)
+    yield collection_dir
+
+    if os.path.isdir(test_dir):
+        shutil.rmtree(test_dir)
+
+
+@pytest.mark.parametrize('collection_skeleton', [
+    ('ansible_test.my_collection', None),
+], indirect=True)
+def test_collection_default(collection_skeleton):
+    meta_path = os.path.join(collection_skeleton, 'galaxy.yml')
+    assert os.path.exists(meta_path)
+
+    with open(meta_path, 'r') as galaxy_meta:
+        metadata = yaml.safe_load(galaxy_meta)
+
+    for item in ['namespace', 'name', 'version', 'authors', 'description', 'license', 'tags', 'dependencies',
+                 'repository', 'documentation', 'homepage', 'issues']:
+        assert item in metadata, 'unable to find {0}'.format(item)
+
+    assert metadata.get('namespace', '') == 'ansible_test'
```

Comment:
Previous check (line 492) suggests that the key has to always exist, no need for fallbacks:
```suggestion
    assert metadata['namespace'] == 'ansible_test'
```
In fact, it should fail loudly if not exists.